css/apps/media.scss

```diff
@@ -273,4 +273,33 @@
 			}
 		}
 	}
+
+	.file-picker__actions {
+		flex-direction: row-reverse;
+	
+		.upload-picker {
+			display: none;
+		}
+	
+		button {
+			&:disabled {
+				&>.button-vue__wrapper>.button-vue__icon>.material-design-icon.image-plus-icon {
```

Comment:
no need for such long selectors, just .material-design-icon.image-plus-icon will be enough

Reply:
Fixed